memstore/ingestion.go

```diff
@@ -206,6 +206,9 @@ func (shard *TableShard) insertPrimaryKeys(primaryKeyColumns []int, eventTimeCol
 				return nil, nil, nil, utils.StackError(err, "Event time for row %d is null", row)
 			}
 			primaryKeyEventTime = upsertBatch.ArrivalTime
+			if primaryKeyEventTime < shard.LiveStore.ArchivingCutoffHighWatermark {
```

Comment:
they can be combined into the previous line
```
if  primaryKeyEventTime = upsertBatch.ArrivalTime; primaryKeyEventTime < shard.LiveStore.ArchivingCutoffHighWatermark {
```

Reply:
done